spec/hipchat_api_v2_spec.rb

```diff
@@ -317,6 +319,24 @@
     end
   end
 
+  describe "#user_update" do
+    include_context "HipChatV2"
+
+  let(:user_update) {
+      {
+        "name" => "Foo Bar",
+        "presence" => {"status" => "Away", "show"=>"away"},
+        "mention_name" => "foo",
+        "email" => "foo@bar.org"
+      }
+    }
+  
+    
+    it "successfull" do
+      mock_succesful_user_update(user_update)
```

Comment:
Does this spec test anything without an `expect`?

Also, I don't think the `user_update` value is actually used in `mock_successful_user_update`.